open_spiel/games/backgammon.h

```diff
@@ -264,6 +264,9 @@ class BackgammonGame : public Game {
 
   // There is arbitrarily chosen number to ensure the game is finite.
   int MaxGameLength() const override { return 1000; }
+  // It is possible to have only chance nodes in the game. As the game
```

Comment:
This is not possible in backgammon, iiuc

Reply:
Then maybe something is wrong with the implementation. I was able to expand the tree to history [0, 0, 0, ... ] = [0] * 1000

Reply:
This is strictly true because of how the tie cases are currently handled for determining first player. We should just change this to not roll doubles for determining who starts. I will fix it this week.

But that was not the reason for choosing MaxGameLength here, at least not initially. That was because in Backgammon there is once chance move per decision move. 

So let's keep this as is for now snd when I fix the impl, I will remove/change this comment